src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams, HttpHeaders } from '@angular/common/http';
+
+import { Observable, of } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+import { User } from './auth';
+
+@Injectable()
+export class AuthService {
+  authUrl = 'https://js-band-store-api.glitch.me/signin';
```

Comment:
better to store urls as constant inside auth module folder (separate file called constants)